collections/retail/custom-engagements/templates/apple-pay-launch-template/content-schema.json

```diff
@@ -40,7 +40,7 @@
         "picture": {
           "title": "Image",
           "type": "file",
-          "default":"{{contextRoot}}/assets/apple-pay.png",
+          "default":"{{contextRoot}}/assets/dark.png",
```

Comment:
I don't see this `dark.png` in the assets folder. Did you test it in the internal repository?